doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1028 @@
+        return speed
+
+    def __get_speed_intersection(self, behavior: str) -> float:
+        speed = 0.0
+        if behavior == bs.int_app_init.name:
+            speed = bs.int_app_init.speed
+        elif behavior == bs.int_app_green.name:
+            speed = bs.int_app_green.speed
+        elif behavior == bs.int_app_to_stop.name:
+            speed = self.__calc_speed_to_stop_intersection()
+        elif behavior == bs.int_wait.name:
+            speed == bs.int_wait.speed
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)